### ISSUE TITLE
[3.7 com_fields] Create new Field Group on the fly

### ISSUE THREAD

**Issue by franz-wohlkoenig** [joomla/joomla-cms#13342]
### Steps to reproduce the issue
Set a new Field (User, Contact or Article) and try to set a new Field Group (like in "Articles" setting a new Category).
### Expected result
Creating a new Group in Dropdown "Field Group".
### Actual result
Dropdown shows existing Field Groups, no "new" Possibility like in "Articles".
### System information (as much as possible)
Joomla! 3.7.0-alpha1
PHP 7.0.4
MySQLi 5.5.53-0+deb7u1

**Comment by laoneo** [comment]
This is a regression from #13019. This feature got lost as we went away from categories. Also the state links are missing 
![image](https://cloud.githubusercontent.com/assets/251072/21450496/fcfe79fa-c8f7-11e6-9483-3541571a310c.png)

**Comment by Bakual** [comment]
Yep I didn't include them as I thought they can be added again even in a future release and they are only convenient features. So it wasn't important to me for the moment.

**Comment by ggppdk** [comment]
This is not a bug or anything
it is a non-important feature, **that does not exist**
(if we were still using categories as field groups then you could call this a bug, if it did not work),

Not worth putting any effort on it now, it is for the future

**Comment by franz-wohlkoenig** [comment]
As User i expect it like other Parts (Article/Categorie) of Joomla!; if not, it looks callow.

**Comment by laoneo** [comment]
> Not worth putting any effort on it now

Please stop judging where it is worth to put effort in and where not. If it is a required feature for people then we should accept that.

User groups have it, categories do have it, so field groups should definitely have it as well. @pe7er do you think have time to have a look on it, if there would be a way to implement that functionality for field groups as well?

**Comment by Bakual** [comment]
I think what @ggppdk meant is that this can be implemented in a later release as well as it has no B/C implications at all. So there is no hurry and it is not a critical feature at all. He is also right that it is not a bug, this is a feature request.

That being said, of course it would be a nice feature and if someone wants to take the time to implement it, go ahead!

**Comment by brianteeman** [comment]
It would be great to get the count links implemented for J4

**Comment by alikon** [comment]
i'll move it on discussion as it is not a bug but a new feature request
